src/gui/GuiWindow.zig

```diff
@@ -46,6 +46,7 @@ const RelativePosition = union(enum) {
 
 const snapDistance = 3;
 
+gameMenuNeed:bool = false,
```

Comment:
It is the other way around! The "GameMenu Needs" this window to close.
Basicly: As long one of the needed window is open, the gamemenu can't get closed.
so pressing c e opens two windows, pressing c closes one, but because e is still open, the gameMenu need this window to be closed too

Reply:
That is a tough naming problem. Maybe `cantCloseGameMenuIfOpen`?
Either way I think it would be great to have a comment with some further explanation.

Reply:
`blocksMenuExit`